packages/http/src/util.js

```diff
@@ -195,4 +196,15 @@ export {
    * console.log(id); // Output:'3f4e254e-8f6f-4f8b-9651-1c1c262cc83f'
    */
   uuid,
+  /**
+   * Returns a key, value pair in an array.
```

Comment:
This is the blocking problem: we cannot manually duplicate these docs across all adaptors.

Reply:
These docs need removing :)